Mathlib.lean

```diff
@@ -2397,6 +2397,7 @@ import Mathlib.Tactic.Use
 import Mathlib.Tactic.WLOG
 import Mathlib.Tactic.Zify
 import Mathlib.Tactic.Zify.Attr
+import Mathlib.Tactic.extractGoal
```

Comment:
I would expect the filename to still begin with a capital letter

Reply:
Changed, thanks!